### PR TITLE
Add Peplink deploy hook

### PR DESCRIPTION
**Pull request by markchalloner**

### CODE REVIEW

**Comment by Neilpang** [comment]
please add your usage here:
https://github.com/acmesh-official/acme.sh/wiki/deployhooks

And create an issue for future bugs: 
https://github.com/acmesh-official/acme.sh/wiki/DNS-API-Dev-Guide#10-please-create-a-new-issue-for-future-bugs

**Comment by markchalloner** [comment]
@Neilpang see:

- https://github.com/acmesh-official/acme.sh/issues/3353
- https://github.com/acmesh-official/acme.sh/wiki/deployhooks#24-deploy-the-cert-into-a-peplink-router

File: deploy/peplink.sh [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  _getdeployconf PEPLINK_Scheme
+  _getdeployconf PEPLINK_Port
+
+  # Default vaules for certificate type, scheme, and port
+  [ -n "${PEPLINK_Certtype}" ] || PEPLINK_Certtype="webadmin"
+  [ -n "${PEPLINK_Scheme}" ] || PEPLINK_Scheme="https"
+  [ -n "${PEPLINK_Port}" ] || PEPLINK_Port="443"
+
+  # Get the certificate type, but don't save it until we verify it exists and is supported
+  _savedeployconf PEPLINK_Scheme "$PEPLINK_Scheme"
```
Comment (Neilpang) [inline deploy/peplink.sh:68]: move these 2 lines ahead of ` [ -n "${PEPLINK_Certtype}" ] || PEPLINK_Certtype="webadmin"`

you don't need to save PEPLINK_Scheme and PEPLINK_Port if they are not provided by the user.

**Comment by markchalloner** [comment]
@Neilpang , thank you and thanks for the great software!

**Pull request merged**